source/_components/sensor.ebusd.markdown

```diff
@@ -0,0 +1,147 @@
+---
+layout: page
+title: "ebusd"
+description: "The ebusd sensor allows the integration between eBUS heating system and Home Assistant."
+date: 2018-10-01 15:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ebusd.png
+ha_category: Sensor
+ha_iot_class: "Local Polling"
+ha_release: "0.80"
+---
+
+Integration between `ebusd` daemon for communication with eBUS heating systems, and homeassistant using sensor component.
+
+## {% linkable_title Configuration %}
+
+Enable the sensor by adding the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: ebusd
+    host: 127.0.0.1
+    port: 8888
+    name: 'ebusd'
+    circuit: '700'
+    monitored_conditions:
+      - 'MaxFlowTemperatureDesired'
+      - 'MinFlowTemperatureDesired'
+      - 'WaterPressure'
+      - 'PumpStatus'
+      - 'HWTemperatureDesired'
+      - 'HWTimerMonday'
+      - 'HWTimerTuesday'
+      - 'HWTimerWednesday'
+      - 'HWTimerThursday'
+      - 'HWTimerFriday'
+      - 'HWTimerSaturday'
+      - 'HWTimerSunday'
+      - 'Zone1NightTemperature'
+      - 'Zone1DayTemperature'
+      - 'Zone1RoomTemperature'
+      - 'Zone1ActualRoomTemperatureDesired'
+      - 'Zone1TimerMonday'
+      - 'Zone1TimerTuesday'
+      - 'Zone1TimerWednesday'
+      - 'Zone1TimerThursday'
+      - 'Zone1TimerFriday'
+      - 'Zone1TimerSaturday'
+      - 'Zone1TimerSunday'
+      - 'Zone1OperativeMode'
+      - 'PowerEnergyConsumptionThisMonth'
+```
+
+{% configuration %}
+host:
+  description: This is the IP address of your ebus daemon, eg. 127.0.0.1.
+  required: true
+  type: string
+circuit: 
+  description: The heating circuit name to monitor, eg. 700.
+  required: true
+  type: string
+port:
+  description: The port your ebus daemon uses, defaults to 8888.
+  type: integer
+  required: false
+  default: 8888
+name: 
+  description: The name to use when displaying this ebusd instance.
+  type: string
+  required: false
+monitored_conditions:
+  description: array: List of condition to monitor.
```

Comment:
Try to add a quote to this description (since it contains ":").
I'm checking out the code now to build locally and see if it is enough.

Reply:
It should be enough to correct this line. If you use any special characters (such as :, [] etc.) you need to place the text in quotes.
In this case it might be easier to just remove "array: ", since I think it's clear from "map" that it's a list?